hr_shift/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="hr.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//block[@name='work_organization_setting_container']"
+                position="inside"
+            >
+                <setting
+                    company_dependent="1"
+                    help="Set default company's default working start and end day to be used in shifts span"
+                >
+                    <label
+                        for="shift_start_day"
+                        string="Week gos from"
+                        class="o_light_label"
+                    />
+                    <field name="shift_start_day" class="oe_inline" required="1" />
+                    <label for="shift_end_day" string="to" class="ml-2 o_light_label" />
+                    <field name="shift_end_day" class="oe_inline" required="1" />
+                </setting>
+            </xpath>
```

Comment:
This needs to be adjusted better; it doesn't look the same as in v15.

**v15**
<img width="1029" height="234" alt="config-15" src="https://github.com/user-attachments/assets/000901d4-eab1-4464-b698-ff322b100230" />

**v17**:
<img width="1095" height="266" alt="config-17" src="https://github.com/user-attachments/assets/896e9839-5b21-4142-bdf6-949529d184f9" />